tests/Unit/Http/HeadersTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace DDTrace\Tests\Unit\Http;
+
+use DDTrace\Http\Headers;
+use PHPUnit\Framework;
+
+
+final class HeadersTest extends Framework\TestCase
+{
+    public function test_headersMapToColonSeparatedValues_emptyArray()
+    {
+        $this->assertEmpty(Headers::headersMapToColonSeparatedValues([]));
+    }
+
+    public function test_headersMapToColonSeparatedValues_null()
+    {
+        $this->assertEmpty(Headers::headersMapToColonSeparatedValues(null));
```

Comment:
Is there a reason we want to accept `null` here as opposed to `[]`? :)

Reply:
Removing it (as we changed the code in the original function) :)